src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,65 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Long FIRST_ID = 1L;
+    private static final Long SECOND_ID = 2L;
```

Comment:
if I download your app if this will valid data for me. I mean that will be I have driver with id 2?
оскільки id це auto-increment значення то ми не можемо знати, що на іншій машині буде згенеровано те саме значення id і при запуску виникне помилка. Тому звертатися на пряму по id не найкраща ідея, краще передавати щось типу manufacturer1.getId()